### ISSUE TITLE
[BUG] youtube videos are one second short

### ISSUE THREAD

**Issue by B00fie** [futo-org/grayjay-android#1015]
As you can see in the following video:
https://www.youtube.com/watch?v=yzGSixROShA

Upon viewed on official youtube app the last whitening flash is visible. On grayjay-android youtube, white flash is not visible (it is not playing the last second of the video)

**Comment by lucina-da-lucario** [comment]
Yeah, that's very noticeable with one or two second videos. Take the Quake HUP sound effect. 
![Screenshot_20240520_233332_Grayjay](https://github.com/futo-org/grayjay-android/assets/61949068/f237eb69-ee06-4763-b914-9076aa31bc2d)
The video is a second long, but in the app, 
![Screenshot_20240520_233435_Grayjay](https://github.com/futo-org/grayjay-android/assets/61949068/62f5892b-510f-4ac2-86d6-d1e273ac05a2)
it's not a single second long.

**Comment by xephos1one** [comment]
Duplicate of #36

**Comment by B00fie** [comment]
Thanks with mention, I will be following the main issue aswell
